src/literals.py

```diff
@@ -19,6 +19,15 @@
 CHARM_USERS = ["monitor"]
 SERVER_PORT = 5601
 
+DEPENDENCIES = {
+    "osd_upstream": {
+        "dependencies": {},
+        "name": "opensearch-dashboards",
+        "upgrade_supported": ">=2",
+        "version": "2.12",
```

Comment:
FYI - we are at 2.14

Reply:
Yes, I noticed the Opensearch upgrade.

I'm adding tickets to upgrade the Dashboards as well when I'm back.

Reply:
- [DPE-4565](https://warthogs.atlassian.net/browse/DPE-4565) [OpenSearch Dashboards][VM] - Upgrade - Snap v2.13
- [DPE-4566](https://warthogs.atlassian.net/browse/DPE-4566) [OpenSearch Dashboards][VM] - Upgrade - Snap v2.14
- [DPE-4567](https://warthogs.atlassian.net/browse/DPE-4567) [OpenSearch Dashboards][VM] - Upgrade - Upgrade - Charm workload version v2.13 
- [DPE-4568](https://warthogs.atlassian.net/browse/DPE-4568) [OpenSearch Dashboards][VM] - Upgrade - Upgrade - Charm workload version v2.14